src/frontend/src/components/ReactHookEditableList.tsx

```diff
@@ -8,28 +8,51 @@ interface ReactHookEditableListProps {
   register: UseFormRegister<any>;
   append: UseFieldArrayAppend<any, any>;
   remove: UseFieldArrayRemove;
+  title: string;
 }
 
-const ReactHookEditableList: React.FC<ReactHookEditableListProps> = ({ name, ls, register, append, remove }) => {
+const ReactHookEditableList: React.FC<ReactHookEditableListProps> = ({ name, ls, title, register, append, remove }) => {
   return (
     <>
-      {ls.map((_element, i) => {
-        return (
-          <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
-            <TextField required autoComplete="off" sx={{ width: 9 / 10 }} {...register(`${name}.${i}.detail`)} />
-            <IconButton type="button" onClick={() => remove(i)} sx={{ mx: 1, my: 0 }}>
-              <DeleteIcon />
-            </IconButton>
-          </Grid>
-        );
-      })}
+      <Typography variant="h5">{title}</Typography>
+      <Grid item sx={{ my: 1, display: 'flex', flexDirection: 'row', flexWrap: 'wrap' }}>
+        {ls.map((_element, i) => {
+          return (
+            <>
+              <Grid item sx={{ display: 'flex', flexDirection: 'column', marginRight: '20px' }}>
```

Comment:
you can delete the fragment component you're using to wrap everything here, and just make this a grid item with these props: xs={12} md={6} lg={4} (shouldn't need any of the styling)